build/common.legacy.props

```diff
@@ -4,7 +4,7 @@
     <IsNetCoreProject>false</IsNetCoreProject>
   </PropertyGroup>
   <ItemGroup Condition="'$(Shipping)' == 'true' AND '$(IsXPlat)' != 'true'">
-    <PackageReference Include="PdbGit" Version="3.0.41" PrivateAssets="all"/>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-19351-01" PrivateAssets="All"/>
```

Comment:
I think you should move this to common.project.props. 
I think both common.props/common.legacy.props import that.